pkg/apis/config/feature_flags.go

```diff
@@ -118,6 +119,35 @@ const (
 	coscheduleKey                       = "coschedule"
 )
 
+var knownFeatureFlagKeys = sets.NewString(
+	KeepPodOnCancel,
+	EnableCELInWhenExpression,
+	EnableStepActions,
+
+	disableAffinityAssistantKey,
+	disableCredsInitKey,
+	runningInEnvWithInjectedSidecarsKey,
+	awaitSidecarReadinessKey,
+	requireGitSSHSecretKnownHostsKey,
+	enableTektonOCIBundles,
+	enableAPIFields,
+	sendCloudEventsForRuns,
+	enforceNonfalsifiability,
+	verificationNoMatchPolicy,
+	enableProvenanceInStatus,
+	resultExtractionMethod,
+	maxResultSize,
+	setSecurityContextKey,
+	coscheduleKey,
```

Comment:
We recently added a few new feature flag: https://github.com/tektoncd/pipeline/blob/7069889e796b0b12e12a3d1095a5f1a3d98c2739/pkg/apis/config/feature_flags.go#L102C1-L112C32

Please add them to the set as well 😁